spec/controllers/curator/institutions_controller_spec.rb

```diff
@@ -34,6 +34,7 @@
       id_from_auth: location.id_from_auth,
       coordinates: location.coordinates,
       authority_code: location.authority_code,
+      bounding_box: location.bounding_box,
```

Comment:
If I understand correctly, this is needed because the spec is referencing a previously-created geo subject fixture that contains a bounding box, yes?

Reply:
Yes that is correct